custom/tests.yaml

```diff
@@ -3284,17 +3311,28 @@ api:
         return false;
       }
 
+      // The following function is defined to help obtain individual stats types
+      // and entries
+      function getStatByTypeAndEntry(type, entry) {
+        for (var i = 0; i < stats.length; i++) {
+          if (stats[i].type === type
+            && Object.prototype.hasOwnProperty.call(stats[i], entry)
+            && stats[i][entry] !== null) {
+            return stats[i];
+          }
+        }
+        return false;
+      }
+
```

Comment:
What do you think about merging this with `getStatByType()` and changing the behavior depending on if the `entry` variable is present?

Reply:
Yeah, it is a good idea. I applied your suggestion and updated the PR.